RecoVertex/PixelVertexFinding/plugins/alpaka/clusterTracksByDensity.h

```diff
@@ -76,7 +76,11 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::vertexFinder {
     // fill hist (bin shall be wider than "eps")
     for (auto i : cms::alpakatools::uniform_elements(acc, nt)) {
       int iz = static_cast<int>(zt[i] * 10.f);  // valid if eps <= 0.1
-      iz = std::clamp(iz, INT8_MIN, INT8_MAX);
+      // Equivalent of iz = std::clamp(iz, INT8_MIN, INT8_MAX)
+      // which doesn't compile with gcc14 due to reference to __glibcxx_assert
+      // See https://github.com/llvm/llvm-project/issues/95183
+      int tmp_max = std::max<int>(iz, INT8_MIN);
+      iz = std::min<int>(iz, INT8_MAX);
```

Comment:
tmp+max is unused....

Reply:
Thanks!

Reply:
maybe we should use NVIDIA implementation for cuda backends
https://nvidia.github.io/cccl/libcudacxx/

Reply:
https://github.com/NVIDIA/cccl/blob/3786a088f986179b30bff006bd0d78853e5013dc/libcudacxx/include/cuda/std/__algorithm/clamp.h

Reply:
True, but we will also need to implement workaround for ROCm (I think it has the same issue)

Reply:
did you check with hipcc?  My understanding was that llvm took care of those